src/configurator/index.js

```diff
@@ -20,6 +20,7 @@ const defaultOptionsCLIonly = {
   production: false,
   static: false,
   server: false,
+  open: false,
```

Comment:
Also maybe this option should not be "cli only" so we can adjust from package.json